arches/app/etl_modules/branch_excel_importer.py

```diff
@@ -24,15 +24,27 @@
 
 
 class BranchExcelImporter(BaseImportModule):
-    def __init__(self, request=None, loadid=None, temp_dir=None):
-        self.request = request if request else None
-        self.userid = request.user.id if request else None
+    def __init__(self, request=None, loadid=None, temp_dir=None, params=None):
+        self.loadid = request.POST.get("load_id") if request else loadid
+        self.userid = (
+            request.user.id if request else User.objects.filter(is_superuser=True)[0].id
+        )
```

Comment:
I think instead of finding the first superuser, it would be better to require that a username be passed into the command - maybe like psql (-U, --user).

Reply:
For the time being you could also just use ```settings.DEFAULT_RESOURCE_IMPORT_USER["userid"]```, but eventually it would be nice to optionally pass a user to the command.